React/src/components/EditProfileButton.tsx

```
@@ -0,0 +1,22 @@
+import { RedirectRequest } from "@azure/msal-browser";
+import { useMsal } from "@azure/msal-react";
+import Button from "controls/Button";
+import { b2cPolicies } from "../authConfig";
+
+const EditProfileButton = () => {
+    const { instance } = useMsal();
+
+    const handleEditProfile = async () => {
+        try {
+            await instance.loginRedirect({ ...b2cPolicies.authorities.editProfile} as RedirectRequest)
+        } catch(e) {
+            console.error(e);
```

Nên bỏ thói quen này, ngay cả khi đang dev. Khi em catch nghĩa là em đang expect 1 error gì đó thì nên catch specific error đó và có some handling, at least là có comment để người khác biết đang muốn handle error gì.

Try catch all và chỉ write ra kiểu này sẽ khiến cho issue ngày càng khó detect và system sẽ rất unstable. Console.log nó chỉ print ra cho người dùng chứ mình ko track dc nên mình vừa polute F12 console của người dùng mà mình cũng sẽ ko biết dc là đang có issue trong prod.

Còn nếu intentionally muốn ignore error trong 1 số trường hợp thì bắt buộc phải để comment để người khác biết là cố ý.